server/services/DEPRECATEnotificationService.js

```diff
@@ -3,72 +3,55 @@ const knex = require('knex')
 const db = require('../database')
 
 const DEPRECATEnotificationService = {
-  // this is only used to count the number of notifications in calendar.ejs, and we have two other methods in this class that do the same thing.
+  // this is only used to count the number of notifications in calendar.ejs
   async getShiftNotifications(quantumId) {
     return db
-      .select(
-        knex.raw(
-          `"DESCRIPTION" as "Description", "LAST_MODIFIED_DATE_TIME" as "LastModifiedDateTime", "PROCESSED" as "Processed"`,
-        ),
-      )
-      .from('SHIFT_NOTIFICATION')
-      .where('QUANTUM_ID', '=', quantumId.toLowerCase())
+      .select('quantum_id', 'last_modified')
+      .from('shift_notification')
+      .where('quantum_id', '=', quantumId.toLowerCase())
       .union(
         db
-          .select(
-            knex.raw(
-              `"DESCRIPTION" as "Description", "LAST_MODIFIED_DATE_TIME" as "LastModifiedDateTime", "PROCESSED" as "Processed"`,
-            ),
-          )
-          .from('SHIFT_TASK_NOTIFICATION')
-          .where('QUANTUM_ID', '=', quantumId.toLowerCase()),
+          .select('quantum_id', 'last_modified')
+          .from('shift_task_notification')
+          .where('quantum_id', '=', quantumId.toLowerCase()),
       )
-      .orderBy('LastModifiedDateTime', 'desc')
       .catch((err) => {
         throw err
       })
   },
 
   getShiftNotificationsPaged(quantumId, offset, perPage) {
     return db
-      .select(
-        knex.raw(
-          `"DESCRIPTION" as "Description", "LAST_MODIFIED_DATE_TIME" as "LastModifiedDateTime", "PROCESSED" as "Processed"`,
-        ),
-      )
-      .from('SHIFT_NOTIFICATION')
-      .where('QUANTUM_ID', '=', quantumId.toLowerCase())
+      .select(knex.raw(`description AS "Description", last_modified AS "LastModifiedDateTime"`))
```

Comment:
Why has `processed` been dropped from here?

Reply:
because, we don't actually use it for the data, its used to get amount of rows, so it needs enough to create unique rows, something, maybe knex or merges them down to a unique set  if you just do quantumId, it can go back in it was just quicker to delete it than retype it in lower case. except now I've had to type all this out 😄 .

Reply:
we could probably turn it into a count, but its only going to exist for another 4 days or so.

Reply:
Okay :)